pkg/hostpath/hostpath_test.go

```diff
@@ -95,318 +95,282 @@ func TestCreateHostPathVolumeBadAccessType(t *testing.T) {
 }
 
 func TestCreateDeleteConfigMap(t *testing.T) {
-	readOnly := []bool{true, false}
```

Comment:
in my attempt to change these tests, I was wondering if it wouldn't be interresting to keep this for loop with 2 attempts; so we are sure that a creation then a deletion then a re-creation works well.
That would change the for loop with an i++ counter instead, but that would keep the test valid and valuable; wdyt ?

Reply:
Sorry no, I don't think such a permutation has value for any of these tests.  The volume won't even be provisioned when the kubelet asks for it if readOnly is not set to true with these changes.  That is the requirement we have had to establish, the pattern from upstream we are following, in order to get the SELinux labels correct.